cli/pkg/cmdutil/helper.go

```diff
@@ -439,6 +443,13 @@ func (h *Helper) OpenRuntimeClient(ctx context.Context, org, project string, loc
 	return rt, instanceID, nil
 }
 
+func (h *Helper) GitHelper(client *client.Client, org, project, localPath string) *GitHelper {
```

Comment:
Ah I see you already added this – do you need the `NewGitHelper` to be exported then? It'd be great if this was the only entrypoint for getting a Git helper.